### ISSUE TITLE
Blue 3dx Zone Please Add To The Unistore (Request)

### ISSUE THREAD

**Issue by Blue-3dx** [Universal-Team/db#277]
### Title

Blue 3dx Zone

### System(s)

3DS

### Categories

Luma3DS

### Unique ID(s)

none

### Image link

none 

### Icon link

none

### Long description

Blue 3dx Zone Is a custom Nintendo Zone The This Project Is Not Finished And Still needs stuff

### Screenshots

![zone_rogo_C](https://github.com/user-attachments/assets/d1e0488c-42af-4cf9-b6ec-4ca052cc43ce)

### Author

Blue 3dx(Rin)

### Short description

Blue 3dx Zone Is a custom Nintendo Zone The This Project Is Not Finished And Still needs stuff

### Source code link

https://drive.google.com/file/d/1dDTZ74gvjmMaf563Z2DMlaprCzfHJ-Ae/view?usp=sharing

### Creation date

September 21 2024

### Last update date

September 22 2024

### Website link

none

### Wiki link

none

### Download page link

https://drive.google.com/file/d/1dDTZ74gvjmMaf563Z2DMlaprCzfHJ-Ae/view?usp=sharing

### Version number

1.0

### Version title

Blue 3dx Zone 1.0

### Other notes

So basically Make 3 Download Options
1.USA
2.JP
3.EUR
when you download Blue 3dx zone a folder should be created:
1.USA=0004001000021B00
2.JP=0004001000020B00
3.EUR=0004001000022B00
(all these folder should be in luma>titles>one of those ids)
and the romfs.zip should be extracted inside one of these folder depending on the region

### Developer permission

- [x] I am the app developer or have permission from the app developer to make this request; my request is compliant with the [Universal-DB guidelines](https://github.com/Universal-Team/db/blob/da0f7076fa5632294af0c597efe315bcc86acab4/CONTRIBUTING.md#app-requests)

**Comment by Epicpkmn11** [comment]
This appears to be very much a WIP and is not open source on GitHub (I don't think Universal-Updater is capable of downloading from Google Drive, and I'm hesitant to take that as a source if it can)

I'm closing this issue, but you can reopen if you have a more complete app in the future.
